plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1088,7 +1205,7 @@ def main():
     try:
         proxmox = ProxmoxAPI(api_host, verify_ssl=validate_certs, **auth_args)
         global PVE_MAJOR_VERSION
-        PVE_MAJOR_VERSION = 3 if proxmox_version(proxmox) < LooseVersion('4.0') else 4
+        PVE_MAJOR_VERSION = 3 if proxmox_version(proxmox) < LooseVersion('4.0') else proxmox_version(proxmox).version[0]
```

Comment:
This avoids calling `proxmox_version(proxmox)` twice:
```suggestion
        version = proxmox_version(proxmox)
        PVE_MAJOR_VERSION = 3 if version < LooseVersion('4.0') else version.version[0]
```

Reply:
thought about the same while typing :)